health/health.d/net.conf

```diff
@@ -110,6 +110,34 @@ families: *
     info: the ratio of outbound dropped packets vs the total number of sent packets of the network interface, during the last 10 minutes
       to: sysadmin
 
+# -----------------------------------------------------------------------------
+# interface errors
+
+template: interface_inbound_errors
+      on: net.errors
```

Comment:
Hi @lassebm ,

Netdata applies templates only for `contexts` and we do not have any context named `net.errors`. To find a `context` name you can do the following request `curl -o charts.txt http://localhost:19999/api/v1/charts?all`.

For these two alarms that you are adding, you should use `ipv4.errors` that is the context for ipv4 error chart.

Best regards!

Reply:
The added alarms are for interface errors, not IPv4. See https://github.com/netdata/netdata/blob/master/collectors/freebsd.plugin/freebsd_getifaddrs.c#L525.

Reply:
I am sorry for the confusion, my laptop does not have these charts, so I confused with traffic errors :(